galaxy/importer/models.py

```diff
@@ -71,6 +72,60 @@ def __init__(self, branch, commit, format, contents,
         self.quality_score = quality_score
 
 
+@attr.s(frozen=True)
+class CollectionInfo(object):
+    """Represents collection_info metadata in collection manifest."""
+
+    namespace = attr.ib()
+    name = attr.ib()
+    version = attr.ib()
+    license = attr.ib()
+    description = attr.ib(default=None)
+
+    repository = attr.ib(default=None)
+    documentation = attr.ib(default=None)
+    homepage = attr.ib(default=None)
+    issues = attr.ib(default=None)
+
+    authors = attr.ib(factory=list)
+    tags = attr.ib(factory=list)
+    readme = attr.ib(default='README.md')
```

Comment:
We should not make any assumptions about `METADATA.json`.
It's `mazer` responsibility to generate correct metadata and put it into artifact.
So if there is any readme file available, it must be referenced in `METADATA.json`, otherwise collection doesn't have a readme at all.

Reply:
Agreed, updated